patterns/components/breadcrumb.php

```diff
@@ -1,7 +1,84 @@
 <?php
-  if (function_exists('yoast_breadcrumb')) {
-  yoast_breadcrumb('
-    <nav class="breadcrumbs" role="navigation"><ul class="breadcrumbs__list">','</ul></nav>
-  ');
+function wordpress_breadcrumbs()
+{
+  $name = 'Home'; //text for the 'Home' link
+  $currentBefore = '<li class="breadcrumbs__list-item font--secondary--xs upper dib"><a class="breadcrumbs__link can-be--white">';
+  $currentAfter = '</a></li>';
+  $li_class = 'breadcrumbs__list-item font--secondary--xs upper dib';
+  $link_class = 'breadcrumbs__link can-be--white';
+  if (!is_home() && !is_front_page() || is_paged()) {
+    echo '<nav class="breadcrumbs" role="navigation"><ul class="breadcrumbs__list">';
+    global $post;
+    $home = get_bloginfo('url');
+    echo '<li class="' . $li_class . '"><a class="' . $link_class . '" href="' . $home . '">' . $name . '</a></li>';
+    if (is_category()) {
+      global $wp_query;
+      $cat_obj   = $wp_query->get_queried_object();
+      $thisCat   = $cat_obj->term_id;
+      $thisCat   = get_category($thisCat);
+      $parentCat = get_category($thisCat->parent);
+      if ($thisCat->parent != 0)
```

Comment:
Please use standard curly brackets for conditional statements (and all control structures) when in writing in all `PHP` files. Alternative syntax (`endif;`, `endwhile;`, `endfor;`, `endforeach;`, etc) for control structures should still be used when in mixed `HTML` and `PHP`.